notion/store.py

```diff
@@ -1,7 +1,8 @@
 import json
 from threading import Thread
 import uuid
-from collections import defaultdict, Callable
+from collections import defaultdict
+from collections.abc import Callable
```

Comment:
```suggestion
from typing import Callable
```

That's what I meant originally. I've imported `Callable` from `collections` by mistake.

Reply:
Yep, sure:)